src/realm/util/file_mapper.cpp

```diff
@@ -199,6 +243,17 @@ void remove_mapping(void* addr, size_t size)
 
 void* mmap_anon(size_t size)
 {
+#ifdef _WIN32
+    HANDLE hMapFile;
+    LPCTSTR pBuf;
+
+    hMapFile = CreateFileMapping(INVALID_HANDLE_VALUE, NULL, PAGE_READWRITE, 0, DWORD(size), nullptr);
```

Comment:
On 64 bit OS you need to extract the high order DWORD from `size` as the 4th argument otherwise we may have problems creating mappings > 2GB.